Event-Triggered Integration as a BYOC component tutorial (#781)
wso2/docs-choreo-dev: en/docs/get-started/tutorials/create-your-first-event-triggered-integration.md

 
    - Use an existing RabbitMQ instance or start a new [RabbitMQ](https://www.rabbitmq.com/download.html) instance on a server that can be accessed via the internet. Obtain the `username`, `hostname`, `password`, and `vhost` from the RabbitMQ instance to use later as environment variables. 

!!! tip "RabbitMQ instance"

>     You can use RabbitMQ services such as [CloudAMQP](https://www.cloudamqp.com/) to set up RabbitMQ instances easily.

Reviewer: ```suggestion
    You can easily set up RabbitMQ instances by utilizing services like [CloudAMQP](https://www.cloudamqp.com/).
```
